packages/models-library/src/models_library/api_schemas_webserver/wallets.py

```diff
@@ -74,4 +73,9 @@ class PaymentTransaction(OutputSchema):
     comment: str = FieldNotRequired()
     created_at: datetime
     completed_at: datetime | None
+    # SEE PaymentTransactionState enum
```

Comment:
why not an enum actually?

Reply:
because is in the postgres library. I have a test that syncs them